cli/README.md

```diff
@@ -15,11 +15,12 @@ Running the tests requires [docker](https://docs.docker.com/get-docker/) to be i
 
 We recommend installing in a project specific virtual environment created using a environment
 management tool such as [conda](https://docs.conda.io/en/latest/) or
-[virtualenv](https://virtualenv.pypa.io/en/latest/).
+[virtualenv](https://virtualenv.pypa.io/en/latest/). See detaled insctructions [here](virtual-environment.md)
```

Comment:
if we want to have some virtual env documentation then I thin this should go into the `docs/dev` directory

Reply:
resolved here [917f8c5](https://github.com/UCLH-Foundry/PIXL/pull/423/commits/917f8c5ab462f2df25161b795c3d6ddbabc377d9)